Sources/SwiftSyntax/Documentation.docc/Tutorials/SwiftSyntax By Example.tutorial

```diff
@@ -171,7 +171,7 @@
       }
       
       @Step {
-        Then use the `withStatements` method to create a source file
+        Then use the `with*` method to create a source file
```

Comment:
I think we should still mention the key path here I would suggest the following. `<#some text#>` is Xcode’s way of representing placeholders.

```suggestion
        Then use `with(\.statements, <#new statements#>)*` method to create a source file
```

Reply:
I see! let me merge your suggestion.